cmd/update-main_test.go

```diff
@@ -17,44 +17,64 @@
 package cmd
 
 import (
+	"errors"
 	"fmt"
 	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
 	"os"
+	"os/exec"
 	"runtime"
 	"testing"
 	"time"
 )
 
+func goBinModTime(goBinPath string, t *testing.T) (modTime time.Time) {
+	goBinAbsPath, err := exec.LookPath(goBinPath)
+	if err != nil {
+		t.Fatal(err)
+	}
+	fi, err := os.Stat(goBinAbsPath)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return fi.ModTime()
+}
+
 func TestGetCurrentReleaseTime(t *testing.T) {
 	minioVersion1 := time.Now().UTC().Format(time.RFC3339)
 	releaseTime1, _ := time.Parse(time.RFC3339, minioVersion1)
 
 	minioVersion2 := "DEVELOPMENT.GOGET"
-	tmpfile, err := ioutil.TempFile("", "get-current-release-time-testcase")
+	tmpfile1, err := ioutil.TempFile("", "get-current-release-time-testcase-1")
 	if err != nil {
 		t.Fatalf("Unable to create temporary file. %s", err)
 	}
-	defer os.Remove(tmpfile.Name())
-	minioBinaryPath2 := tmpfile.Name()
-	fi, err := tmpfile.Stat()
+	defer os.Remove(tmpfile1.Name())
+
+	minioBinaryPath3 := "go"
+	if runtime.GOOS == globalWindowsOSName {
+		minioBinaryPath3 = "go.exe"
+	}
+	releaseTime3 := goBinModTime(minioBinaryPath3, t)
+
+	minioBinaryPath2 := tmpfile1.Name()
+	fi, err := tmpfile1.Stat()
 	if err != nil {
 		t.Fatalf("Unable to get temporary file info. %s", err)
 	}
-	if err = tmpfile.Close(); err != nil {
+	if err = tmpfile1.Close(); err != nil {
 		t.Fatalf("Unable to create temporary file. %s", err)
 	}
 	releaseTime2 := fi.ModTime().UTC()
```

Comment:
It would be nice to keep `minioVersion2`, `minioBinaryPath2` and `releaseTime2` init statements grouped ie `minioVersion3` and `minioBinaryPath3` and `releaseTime3` are inserted in between